app/js/lib/reducers/form_content.js

```diff
@@ -25,6 +35,32 @@ function stateContainsUnknownKeys( state ) {
 	return !_.isEmpty( getInvalidKeys( state ) );
 }
 
+function clearFieldsIfAddressTypeChanges( newState, payload ) {
+	if ( payload.contentName !== 'addressType'  ) {
+		return;
+	}
+	switch ( payload.value ) {
+		case 'privat':
```

Comment:
I mistakenly put `privat` in here because that's the value from the `title` attribute of the form field. I'll change that in a future commit.

For backwards compatibility in the backend export scripts we still need the German address types.